bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/Arena.java

```diff
@@ -157,11 +157,11 @@ public class Arena implements IArena {
      * temp stats. some of them use player name as key to keep names of players who left. at checkWinners for example.
      * Those maps are not used for db stats but is for internal use only.
      */
-    private HashMap<String, Integer> playerKills = new HashMap<>();
-    private HashMap<Player, Integer> playerBedsDestroyed = new HashMap<>();
-    private HashMap<Player, Integer> playerFinalKills = new HashMap<>();
-    private HashMap<Player, Integer> playerDeaths = new HashMap<>();
-    private HashMap<Player, Integer> playerFinalKillDeaths = new HashMap<>();
+    private final HashMap<String, Integer> playerKills = new HashMap<>();
+    private final HashMap<Player, Integer> playerBedsDestroyed = new HashMap<>();
+    private final HashMap<Player, Integer> playerFinalKills = new HashMap<>();
+    private final HashMap<Player, Integer> playerDeaths = new HashMap<>();
+    private final HashMap<Player, Integer> playerFinalKillDeaths = new HashMap<>();
```

Comment:
Whats the gain of changing this?

Reply:
> Whats the gain of changing this?



> Whats the gain of changing this?

There was an NPE bug when getting the value of this variable. I fixed this error by adding final code.